README.md

```diff
@@ -36,16 +82,18 @@ particular, we are making an effort to maintain a high degree of
 
 To achieve this we use a common set of docker base files with
 overrides for any values that are required to differ for local vs.
-deployment environments. The file naming convention assumed here
-is:
-
-file                          | contents
------------------------------ | ----------------------------------------------------------------------------------------------------
-`docker-compose.yml`          | compose definitions that are shared between all environments
-`local.override.yml`          | compose definitions required exclusively in a local docker environment
-`docker-compose.ecs.yml`      | compose definitions required for deployment to AWS ECS
-`.env`                        | environment variables injected into the compose file, but not automatically visible in containers
-`.secrets`                    | secure information not to be added to source control
+deployment environments. We create 3 files for each service in the templates
+directory. One for the base, then a local override and a ecs override. These 
+files are composed together to create the compose file used for development or
+deploment as needed.
+
+file                              | contents
+--------------------------------- | ----------------------------------------------------------------------------------------------------
+`blacklight-compose.yml`          | compose definitions that are shared between all environments
+`blacklight-compose.local.yml`    | compose definitions required exclusively in a local docker environment
+`blacklight-compose.ecs.yml`      | compose definitions required for deployment to AWS ECS
+`.env`                            | No longer used. All env should be in Amazon SSM 
```

Comment:
Should we add any more breadcrumbs for the Amazon SSM here?

Reply:
I agree. there are a set of tickets in the next sprint about SSM.  I left some low hanging fruit here on purpose as I want others to get used to making edits and I really tried to keep SSM out of scope for this work but ultimatly it was going to be 4 ours to use it and 16 hours to not use it. So I think the breadcrumbs are needed but out of scope here.

Reply:
Ok, can you tell me them though?